addon/mixins/copyable.js

```diff
@@ -3,14 +3,22 @@ import DS from 'ember-data';
 
 export default Ember.Mixin.create({
   copyable: true,
-  copy: function(options) {
+  copy: function(options, copied) {
     options = options || {};
+    copied = copied || {};
 
     var _this = this;
     return new Ember.RSVP.Promise(function(resolve) {
 
       var model = _this.constructor;
+
+      var id = _this.get('id');
```

Comment:
Yeah, that was a good one :) Thanks a lot. Hopefully it will get merged
sooner or later.